.github/workflows/lint.yml

```diff
@@ -12,7 +12,6 @@ jobs:
   lint:
     name: Lint the code
     runs-on: ubuntu-latest
-    container: golang:1.22
```

Comment:
no particular reason, I just wanted less place to maintain the version at. this way, it's only defined in `go.mod` file and nowhere else. 😊